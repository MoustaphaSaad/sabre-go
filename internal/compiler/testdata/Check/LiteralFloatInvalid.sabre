package main

func IsValid() float32 {
    return 1e
}
