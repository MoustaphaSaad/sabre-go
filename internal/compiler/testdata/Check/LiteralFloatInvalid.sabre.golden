>> 	    return 1e
>> 	           ^^ 
Error[internal/compiler/testdata/Check/LiteralFloatInvalid.sabre:4:12]: invalid float value
>> 	    return 1e
>> 	           ^^ 
Note[internal/compiler/testdata/Check/LiteralFloatInvalid.sabre:4:12]: strconv.ParseFloat: parsing "1e": invalid syntax
>> 	    return 1e
>> 	           ^^ 
Error[internal/compiler/testdata/Check/LiteralFloatInvalid.sabre:4:12]: incorrect return type 'void', expected 'float32'
